core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ApplicationSummaryInfo.scala

```diff
@@ -26,6 +26,7 @@ case class ApplicationSummaryInfo(
     val sqlMetrics: Seq[SQLAccumProfileResults],
     val jsMetAgg: Seq[JobStageAggTaskMetricsProfileResult],
     val sqlTaskAggMetrics: Seq[SQLTaskAggMetricsProfileResult],
+    val ioMetrics: Seq[IOAnalysisProfileResult],
```

Comment:
why put this in the middle, add it to the end

Reply:
done